README.md

```diff
@@ -94,6 +97,30 @@ True on cache hit (even if the subsequent
 failed) and false on cache miss. See also
 [skipping steps based on cache-hit](https://github.com/marketplace/actions/cache#Skipping-steps-based-on-cache-hit).
 
+## Anonymous image names
```

Comment:
This section is only relevant to a small minority of users, so let's move it below the Permissions section.